doc/JSON/NPCs.md

```diff
@@ -346,35 +346,37 @@ Field | Default messages/snippets | Used for...
 
 ### Special Custom Entries
 
-Certain entries like the snippets above are taken from the game state as opposed to JSON; they are found in the npctalk function parse_tags. They are as follows:
-
-Field | Used for...
----|---
-`<yrwp>` | displays avatar's wielded item
-`<mywp>` | displays npc's wielded item
-`<u_name>` | displays avatar's name
-`<npc_name>` | displays npc's name
-`<ammo>` | displays avatar's ammo
-`<current_activity>` | displays npc's current activity
-`<punc>` | displays a random punctuation from: `.`, `…`, `!`
-`<mypronoun>` | displays npc's pronoun
-`<total_kills>` | total kills of the Player
-`<time_survived>` | time since start of the game
-`<topic_item>` | referenced item
-`<topic_item_price>` | referenced item unit price
-`<topic_item_my_total_price>` | TODO Add
-`<topic_item_your_total_price>` | TODO Add
-`<interval>` | displays the time remaining until restock
-`<u_val:VAR>` | The user variable VAR
-`<npc_val:VAR>` | The npc variable VAR
-`<context_val:VAR>` | The context variable VAR
-`<global_val:VAR>` | The global variable VAR
-`<item_name:ID>` | The name of the item from ID
-`<item_description:ID>` | The description of the item from ID
-`<trait_name:ID>` | The name of the trait from ID
-`<trait_description:ID>` | The description of the trait from ID
-`<spell_name:ID>` | The description of the name from ID
-`<spell_description:ID>` | The description of the trait from ID
+Certain entries like the snippets above are taken from the game state as opposed to JSON; they are found in the npctalk function parse_tags. In dialogue the alpha talker is the avatar and the beta talker is the NPC.
+
+| Field                                         | Uses alpha | Uses beta | Description
```

Comment:
This is a really obnoxious table to read, it should just be one column which lists either "Alpha" or "Beta" or "N/A". Since checking alpha or beta is mutually exclusive, there is no need for two columns.

(The entry for context_val appears to be just plain wrong, since it is a context val.)